src/main/java/org/datavyu/plugins/ffmpeg/FfmpegJavaMediaPlayer.java

```diff
@@ -281,7 +288,7 @@ protected double playerGetDuration() throws MediaException {
     }
 
     @Override
-    protected void playerSeek(double streamTime, int flags) throws MediaException {
+    protected void ffmpegPlayerSeek(double streamTime, int flags) throws MediaException {
```

Comment:
Could we use overloading here instead?  E.g. refactor ffmpegPlayerSeek -> playerSeek

Reply:
Done!